.github/actions/bump-skip/action.yml

```diff
@@ -24,5 +24,5 @@ runs:
           IS_BUMP=yes
         fi
         echo "IS_BUMP=$IS_BUMP"
-        echo ::set-output name=is-bump::$IS_BUMP
+        echo is-bump=$IS_BUMP >> $GITHUB_OUTPUT
```

Comment:
Does this need to be quoted to substitute the value? I forget the exact rules for GHA syntax

Reply:
Not sure, so I kept as-is.  But nothing has failed...

Reply:
a bunch of passing tests made use of non-quoted values.